cedar-policy/src/api.rs

```diff
@@ -1809,6 +1864,70 @@ impl Policy {
     /// If `id` is Some, the policy will be given that Policy Id.
     /// If `id` is None, then "JSON policy" will be used.
     /// The behavior around None may change in the future.
+    ///
+    /// ```
+    /// use cedar_policy::{Policy, PolicyId};
+    /// use std::str::FromStr;
+    ///
+    /// let data = r#"
```

Comment:
can we follow our own best-practice and declare this data using `serde_json::json!` macro instead of as a string literal?

Reply:
updated